### PR TITLE
feat: TE7-T1022: Implement profile screen redux

### PR DESCRIPTION
**Pull request by jigar-s-simform**
implement profile screen redux

### CODE REVIEW

File: app/components/custom-input/CustomInput.tsx [hunk 1]
```diff
@@ -16,22 +17,29 @@ import CustomTextInputType from './CustomInputTypes';
...
   const inputRef = useRef<TextInput>(null);
+  const [passwordShown, setPasswordShown] = useState<boolean>(false);
 
   useImperativeHandle(ref, () => ({
     focus: () => {
       inputRef.current?.focus();
     },
   }));
 
+  const handlePasswordVisibility = () => {
```
Comment (sagar-simformsolutions) [inline app/components/custom-input/CustomInput.tsx]: Add types

File: app/modules/profile/useProfile.ts [hunk 2]
```diff
@@ -28,31 +30,39 @@ const useProfile = ():UseProfileReturnType => {
...
+      }),
+    );
   };
-
   const handleEditOrSave = () => {
     if (editable) formik.handleSubmit();
     else handleEditProfile();
   };
-
   const handleEditProfilePicture = async () => {
```
Comment (sagar-simformsolutions) [inline app/modules/profile/useProfile.ts]: add types

File: app/modules/profile/useProfile.ts [hunk 3]
```diff
@@ -28,31 +30,39 @@ const useProfile = ():UseProfileReturnType => {
...
+        first_name: values.firstName,
+        last_name: values.lastName,
+        avatar: picturePath,
+        email: values.email,
+        password: values.password,
+      }),
+    );
   };
-
   const handleEditOrSave = () => {
```
Comment (sagar-simformsolutions) [inline app/modules/profile/useProfile.ts]: add types

**Review by sagar-simformsolutions — CHANGES_REQUESTED** [review]

File: app/components/custom-input/CustomInput.tsx (hunk 1 above)
Comment (jigar-s-simform) [reply app/components/custom-input/CustomInput.tsx]: resolved

File: app/modules/profile/useProfile.ts (hunk 2 above)
Comment (jigar-s-simform) [reply app/modules/profile/useProfile.ts]: resolved

File: app/modules/profile/useProfile.ts (hunk 3 above)
Comment (jigar-s-simform) [reply app/modules/profile/useProfile.ts]: resolved

**Pull request merged**